src/api/getContext.ts

```diff
@@ -32,6 +32,7 @@ export type PnpmContext = {
   wantedShrinkwrap: Shrinkwrap,
   skipped: Set<string>,
   pendingBuilds: string[],
+  flattenedPkgAliasesById: {[alias: string]: string}
```

Comment:
so the type should be `{[pkgId: string]: string}`, right?

I'd name it rather `hoistedAliasesByPkgId` or just `hoistedAliases`